### ISSUE TITLE
Support chinese serving size

### ISSUE THREAD

**Issue by aleene** [openfoodfacts/openfoodfacts-server#1618]
**Checklist:**

* [ ] Bug Report - The issue you are reporting might be there already. Please search on the [issues tracker](https://github.com/openfoodfacts/openfoodfacts-server/issues) before creating or reporting a issue.

* [ ] Before reporting, please ensure (if you can) that you have tested it with the latest (dev) version of the software.

**Summary:**

Summarize your issue in one sentence (what goes wrong, what did you expect to happen).

**Steps to reproduce:**

How can we reproduce the issue?

**Add system logs:**

Add logcat files here (if you are able to add).

**Expected behavior:**

What did you expect the app to do?

**Observed behavior:**

What did you see instead?  Describe your issue in detail here.

**Comment by hangy** [comment]
I added some Chinese units in 3f4be298dd8c44860a4de708b82f0baab8e31698. However, the source that I found, [claims](http://www.new-chinese.org/lernwortschatz-chinesisch-masseinheiten.html) that "克" is "g", so I'm not quite sure what's actually correct. Maybe a native speaker could weigh in. 🙂

**Comment by aleene** [comment]
You are correct. They seem to add 公, to indicate metric gram. Anyway they are both found on packages

**Comment by aleene** [comment]
Install google translate on your phone and you can also add chinese products to off.

**Comment by aleene** [comment]
By the way support for other scripts might be required as well (see wikidata item).
